flink-scala-shell/src/test/scala/org/apache/flink/api/scala/ScalaShellITCase.scala

```diff
@@ -320,7 +320,7 @@ object ScalaShellITCase {
   @BeforeClass
   def beforeAll(): Unit = {
     // set to different than default so not to interfere with ScalaShellLocalStartupITCase
-    configuration.setInteger(RestOptions.PORT, 8082)
+    configuration.setInteger(RestOptions.PORT, 0)
```

Comment:
this could simply be removed?

Reply:
I don't think so because the `ScalaShellITCase` still uses the `MiniCluster`. I will check whether `ClassRule` also work with Scala companion objects and then we could replace it with `MiniClusterResource`.